apps/backend/src/config/config.service.ts

```diff
@@ -56,7 +56,8 @@ export class ConfigService {
       oidcName: this.get('OIDC_NAME') || '',
       ldap: this.get('LDAP_ENABLED')?.toLocaleLowerCase() === 'true' || false,
       registrationEnabled: this.isRegistrationAllowed(),
-      localLoginEnabled: this.isLocalLoginAllowed()
+      localLoginEnabled: this.isLocalLoginAllowed(),
+      disableUpdateCheck: !!this.get('DISABLE_UPDATE_CHECK')
```

Comment:
This will only check if there's a non-empty string.  Also please propagate moving the 'disable' part to the end of the name.

```
$ node
Welcome to Node.js v18.12.1.
Type ".help" for more information.
> false
false
> 'false'
'false'
> !'false'
false
> !!'false'
true
```

```suggestion
      updateCheckDisabled: this.get('UPDATE_CHECK_DISABLED')?.toLowerCase() === 'true'
```